draft-ietf-doh-dns-over-https-latest.mkd

```diff
@@ -520,6 +528,83 @@ Author:  Paul Hoffman, paul.hoffman@icann.org
 Change controller:  IESG
 ~~~~~
 
+# Privacy Considerations {#PrivacyConsiderations}
+
+{{RFC7626}} discusses DNS Privacy Considerations in both "On the
+wire" (Section 2.4), and "In the server" (Section 2.5) contexts. This
+is also a useful framing for DoH's privacy considerations.
+
+## On The Wire {#OnTheWire}
+
+DoH encrypts DNS traffic and requires authentication of the
+server. This mitigates both passive surveillance {{RFC7258}} and
+active attacks attempting to divert DNS traffic to rogue servers
+({{RFC7626}} Section 2.5.1). DNS over TLS {{RFC7858}} provides
+similar protections, while direct UDP and TCP based transports are
+vulnerable to this class of information leak.
```

Comment:
You talk about active attacks above, so maybe just say "class of attack"

Reply:
agreed